### ISSUE TITLE
Zero-Shot Voice Cloning

### ISSUE THREAD

**Issue by fakerybakery** [huggingface/parler-tts#31]
Hi,
I know this library is primarily for text -> voice but do you know if it would be possible to modify it to accept a speaker embedding and perform zero-shot voice cloning?
Thanks!

**Comment by digisomni** [comment]
Yep, this is what I am after as well. Bark did this, if we have something like that then using this for an assistant becomes 10x easier since any personality can be inserted into it.

**Comment by ylacombe** [comment]
Hey @fakerybakery, thanks for opening the discussion!
The current design is a choice, and we're currently discussing internally if adding zero-shot voice cloning makes sense!

**Comment by johnwick123f** [comment]
+1, it would be very useful for many things. Parler tts sounds very good and it would be great to support cloning voice

**Comment by sanchit-gandhi** [comment]
Hey all - the two fine-tuned checkpoints ([Jenny](https://huggingface.co/parler-tts/parler-tts-mini-jenny-30H) and [Expresso](https://huggingface.co/parler-tts/parler-tts-mini-expresso)) are interesting intermediate artefacts while we wait for the next pre-trained version of the model with consistent voice support

**Comment by mantrakp04** [comment]
I got really good results with zero shot voice cloning using myshell ai's openvoice 2, m also willing to support in any way to make parler tts multi lingual, lmk

**Comment by Hassan-jr** [comment]
any progress on the  zero-shot voice cloning?

**Comment by johnwick123f** [comment]
Yeah sadly I don’t see any progress on voice cloning. There are consistent voices though.

**Comment by arthurwolf** [comment]
I would love to know when this will become available, this would make this the most useful model around by far. Anyone know of a model with this kind of quality **and** voice cloning ability?

**Comment by johnwick123f** [comment]
@arthurwolf You can try cosyvoice, you can prompt it and do voice cloning. It's actually pretty good but I prefer parler-tts in the prompt part, but cosyvoice is amazing in voice cloning and supports text prompting too(albeit, a bit worse then parler-tts).
